GAMES.json

```diff
@@ -1183,12 +1184,17 @@
 	"323720": true,
 	"324070": true,
 	"324120": true,
+	"324260":
+	{
+		"Comment": "Requires Mono"
```

Comment:
Has a bug been filed with the developer? Anything not in the Runtime is supposed to be bundled, distro libs/runtimes shouldn't be used